recipe_scrapers/rezeptwelt.py

```diff
@@ -0,0 +1,55 @@
+from ._abstract import AbstractScraper
+from ._exceptions import SchemaOrgException
+from ._utils import normalize_string
+
+
+class Rezeptwelt(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "rezeptwelt.de"
+
+    def author(self):
+        return normalize_string(self.soup.find("span", {"id": "viewRecipeAuthor"}).text)
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        return self.schema.ingredients()
+
+    def instructions(self):
+        content = self.soup.find("ol", {"itemprop": "recipeInstructions"}).findAll(
+            "div", {"itemprop": "itemListElement"}
+        )
+        res = ""
+        for i in content:
+            steps = i.findAll("span", {"itemprop": "text"})
+            for step in steps:
+                res = res + normalize_string(step.text) + "\n"
```

Comment:
Suggestion: `x = x + ...` can be rewritten as `x += ...`

```suggestion
                res += normalize_string(step.text) + "\n"
```

Reply:
Fixed in 27ea735e